pkg/action/scan.go

```diff
@@ -342,10 +344,19 @@ func recursiveScan(ctx context.Context, c malcontent.Config) (*malcontent.Report
 			}
 			return nil
 		}
+
+		if bar.State().Description != "" {
```

Comment:
This was the cleanest way to check if a `bar` was configured. 

Using reflect did not work and checking if the State was "zero" by using `bar.State() == (progressbar.State{})` when passing in `bar := progressbar.ProgressBar{}` was showing a mismatch that would not evaluate to true. Either way, we'll always set the description when actually using a progress bar.